int8 control_mode
bool e_stop
uint8 gear
uint8 speed
int32 steer
uint8 brake
int32 encoder
uint8 alive